databases/week3/database_week3.sql

```diff
@@ -0,0 +1,267 @@
+create database meal_sharing;
+
+use meal_sharing;
+
+SET NAMES utf8mb4;
+-- creating meal table
+create table `meal`(
+	`id` INT(10) unsigned NOT NULL AUTO_INCREMENT,
+  `title` VARCHAR(255) NOT NULL,
+  `description` TEXT,
+  `location` VARCHAR(255) NOT NULL,
+  `for_when` DATETIME,
+  `max_reservations` INT(3),
+  `price` DECIMAL NOT NULL,
+  `created_date` DATETIME,
+  PRIMARY KEY (`id`)
+) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci;
+
+-- creating reservation table
+create table `reservation`(
+	`id` INT(10) unsigned NOT NULL,
+	`number_of_guests` INT(3),
+	`meal_id` INT(10) unsigned NOT NULL,
+    `created_date` DATETIME,
+   PRIMARY KEY (`id`),
+   CONSTRAINT `fk_meal_id` FOREIGN KEY (`meal_id`) REFERENCES `meal` (`id`) ON DELETE CASCADE
+) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci;
+
+-- creating review table
+create table `review`(
+	`id` INT(10) unsigned NOT NULL,	
+	`title` varchar(255),
+    `description` TEXT,
+    `meal_id` INT(10) unsigned NOT NULL,
+    `stars` INT(1),
+	`created_date` DATETIME,
+  PRIMARY KEY (`id`),
+  CONSTRAINT `fk_meal_review_id` FOREIGN KEY (`meal_id`) REFERENCES `meal` (`id`) ON DELETE CASCADE
+) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci;
+
+ -- inserting objects to each tables
+ALTER TABLE meal RENAME COLUMN max_reservatins TO max_reservations;
+
+ INSERT INTO meal (id, 
+	 title, 
+	 description, 
+	 location, 
+	 for_when, max_reservations,
+	 price, 
+	 created_date) 
+ VALUE(1, 'Shellfish platter', 
```

Comment:
You do not need to specify the primary key here, as the field has already been defined as auto_increment.